test/gtest/ucp/ucp_test.h

```diff
@@ -266,4 +266,22 @@ std::ostream& operator<<(std::ostream& os, const ucp_test_param& test_param);
     UCP_INSTANTIATE_TEST_CASE_TLS(_test_case, self,   "self") \
     UCP_INSTANTIATE_TEST_CASE_TLS(_test_case, tcp,    "tcp")
 
+/**
+ * Instantiate the parameterized test case for all transport combinations
+ * with CUDA memory awareness
+ *
+ * @param _test_case  Test case class, derived from ucp_test.
+ */
+#define UCP_INSTANTIATE_TEST_CASE_CUDA_AWARE(_test_case) \
+    UCP_INSTANTIATE_TEST_CASE_TLS(_test_case, dcx,        "dc_x,cuda_copy") \
```

Comment:
can we do "dc_x,cuda" for example for all cases? (cuda is alias for all cuda transports)

Reply:
@yosefe I think it will cause using `cuda_ipc` for data transfers where we want to use `dc`

Reply:
good point

Reply:
> good point

@yosefe what do you think about moving my PR changes #4641 to the @bureddy's PR in order to test it on his usage of this macro?

Reply:
yes, @bureddy can you pls rebase?